ArduCopter/crash_check.cpp

```diff
@@ -2,7 +2,7 @@
 
 // Code to detect a crash main ArduCopter code
 #define CRASH_CHECK_TRIGGER_SEC         2       // 2 seconds inverted indicates a crash
-#define CRASH_CHECK_ANGLE_DEVIATION_DEG 30.0f   // 30 degrees beyond angle max is signal we are inverted
+#define CRASH_CHECK_ANGLE_DEVIATION_DEG 60.0f   // 60 degrees beyond angle max is signal we are inverted
```

Comment:
This seams like a big change, that has nothing to do with the rest of the PR

Reply:
this is here by mistake, but this is my preferred value :) It needs to be a parameter, indeed

Reply:
Please remove it from this commit then.